File: wk13recap/freqsol.py
#!/usr/local/bin/python3
from collections import Counter
import sys

counters = [Counter(line) for line in sys.stdin]
counter = sum(counters, Counter())
for letter in sorted(counter):
    if letter.isalnum():
        print("'%s' occured %d times" % (letter, counter[letter]))

File: wk13recap/freqsol.pl
#!/usr/bin/perl -w

%freq = ();
while ($line = <STDIN>) {
    @letters = $line =~ /\w/g;
    foreach $l (@letters) {
        $freq{$l}++;
    }
}

foreach $l (sort keys %freq) {
    printf "'%s' occured %d times\n", $l, $freq{$l};
}
